### PR TITLE
Added registry source, and package publish/install

### PR DESCRIPTION
**Pull request by hongde88**
1. Added registry source to mdctl-packages.
2. Added package publish/install to mdctl-cli.

### CODE REVIEW

File: packages/mdctl-cli/lib/package/publish.js [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+                source, registryUrl, registryProjectId, registryToken, client
+              } = params,
+              pkg = new Package(name, '.')
+
+        await pkg.evaluate()
+
+        let srcClient
+
+        if (source === 'registry') {
+          if (!isSet(registryUrl)) {
```
Comment (gastonrobledo) [inline packages/mdctl-cli/lib/package/publish.js:15]: why not adding this checking inside the Registry class?

**Review by gastonrobledo — CHANGES_REQUESTED** [review]

**Comment by gastonrobledo** [comment]
@hongde88 we could also add some tests for these changes.

File: packages/mdctl-cli/lib/package/publish.js (hunk 1 above)
Comment (hongde88) [reply packages/mdctl-cli/lib/package/publish.js:15]: Good catch there. I actually just realized that I added that check inside Registry class already. Let me remove that check from publish.js.

**Comment by hongde88** [comment]
@gastonrobledo I'm wrapping up some unit tests for my changes and will push them later. Thanks!

**Review by gastonrobledo — APPROVED** [review]

**Pull request merged**